services/workers/src/main/resources/schema-synapse-bootstrap-spb.xml

```diff
@@ -56,8 +53,8 @@
 		</property>
 		<property name="startDelay" value="137" />
 		<!-- This delay determines the maximum amount of time before Synapse object 
-			schema changes will appear on a new production stack.  Currently set to one hour. -->
-		<property name="repeatInterval" value="3600000" />
+			schema changes will appear on a new production stack.  Currently set to two minutes. -->
+		<property name="repeatInterval" value="120000" />
```

Comment:
We can leave this at the original 1 hour interval and update the test-context.xml to override this property